packages/components/time-picker/src/common/picker.vue

```diff
@@ -502,8 +503,8 @@ const onClearIconClick = (event: MouseEvent) => {
   if (showClose.value) {
     event.stopPropagation()
     focusOnInputBox()
-    emitInput(null)
-    emitChange(null, true)
+    emitInput(valueOnClear.value)
+    emitChange(valueOnClear.value, true)
```

Comment:
Maybe we should give `valueOnClear` a default value of `null` here to prevent the change from being too large and giving users a result that is inconsistent with expectations.

Reply:
If I set a default value of `valueOnClear`, the config-provider won't take effect. How can I do?

Reply:
Maybe we can let `useEmptyValues` receive a second parameter and support setting a default value.
```ts
const { valueOnClear } = useEmptyValues(props, null)
...
export const useEmptyValues = (
  props: ExtractPropTypes<typeof useEmptyValuesProps>,
  defaultValue?: null | undefined
) => {
  ...
  const valueOnClear = computed(() => {
    // function is used for undefined cause undefined can't be a value of prop
    if (isFunction(props.valueOnClear)) {
      return props.valueOnClear()
    } else if (props.valueOnClear !== undefined) {
      return props.valueOnClear
    } else if (isFunction(config.value.valueOnClear)) {
      return config.value.valueOnClear()
    } else if (config.value.valueOnClear !== undefined) {
      return config.value.valueOnClear
    }
    return defaultValue !== undefined ? defaultValue :  DEFAULT_VALUE_ON_CLEAR
  })
}
```

Reply:
OK, I will do it.